<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
        <rect>
         <x>540</x>
         <y>340</y>
         <width>111</width>
         <height>41</height>
        </rect>
     </property>
     <property name="text">
        <string>下载</string>
     </property>
    </widget>
  <widget class="QTableWidget" name="fileListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>501</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件大小</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>20010</number>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>./</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>Refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>Upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
    <connection>
     <sender>downloadButton</sender>
     <signal>clicked()</signal>
     <receiver>Widget</receiver>
     <slot>Download()</slot>
     <hints>
        <hint type="sourcelabel">
         <x>599</x>
         <y>350</y>
        </hint>
        <hint type="destinationlabel">
         <x>636</x>
         <y>370</y>
        </hint>
     </hints>
    </connection>
 </connections>
 <slots>
  <slot>Refresh()</slot>
  <slot>Upload()</slot>
    <slot>Download()</slot>
 </slots>
</ui>
